ddhcpd/luasrc/lib/gluon/upgrade/500-ddhcpd

```diff
@@ -19,7 +19,7 @@ if site.ddhcpd then
     payload=site.next_node.ip4()
   })
   uci:delete('ddhcpd','broadcast')
-  cidr=tonumber(string.gsub(site:prefix4(),"[%w\.]+/(%d+)","%1"),10)
+  cidr=tonumber(string.gsub(site:prefix4(),"[%w%.]+/(%d+)","%1"),10)
```

Comment:
Afaik this should be `"[%w.]+/(%d+)"`. There is no need to escape characters inside a character set.

Reply:
but . is the variable for any-character. \. should be the period itself. And in lua, you escape with % instead of \

Reply:
> The character `%´ works as an escape for those magic characters. So, '%.' matches a dot; '%%' matches the character `%´ itself. You can use the escape `%´ not only for the magic characters, but also for all other non-alphanumeric characters. When in doubt, play safe and put an escape.

Reply:
Yes, but inside a character set just `.` matches a literal dot

Reply:
done